.github/scripts/riscv/README.md

```diff
@@ -0,0 +1,28 @@
+# Build RISC-V PyTorch From Scratch
+
+Build the image:
+```bash
+docker build -t riscv-pytorch:latest -f .github/scripts/riscv/Dockerfile .
+```
+
+Run the container:
+```bash
+docker run -it --rm riscv-pytorch:latest
+```
+
+Snapshot:
```

Comment:
Hello, does `Snapshot` section intend to verify build result?

Reply:
Yes. We should see similar results in the end.